lib/mutant/config.rb

```diff
@@ -159,13 +112,14 @@ def self.env
         Transform::Exception.new(YAML::SyntaxError, YAML.method(:safe_load)),
         Transform::Hash.new(
           optional: [
-            Transform::Hash::Key.new('coverage_criteria', CoverageCriteria::TRANSFORM),
+            Transform::Hash::Key.new('coverage_criteria', ->(value) { CoverageCriteria::TRANSFORM.call(value) }),
```

Comment:
I "really" like this require sequence:

```
require 'foo'
require 'foo/bar'
```

But we have to access the constant from `foo/bar` here, hence as we are in class scope and the require for `foo/bar' did not happen yet we have to use a lambda indirection.

The alternative would be:

```
require 'foo/bar'
require 'foo'
```
which sucks. For the moment I eat it. Would love of Ruby had a module system ;).

Reply:
> Would love of Ruby had a module system ;)

yeahh... I feel that one!